### ISSUE TITLE
evaluation

### ISSUE THREAD

**Issue by Bill-Ren** [webis-de/ecir21-an-empirical-comparison-of-web-page-segmentation-algorithms#6]
Thanks for a great job. Your paper uses the Webis-WebSeg-20 dataset, which contains 8490 web pages, for the empirical comparison of web page segmentation algorithms. I would like to ask if you divide the training set and the test set when evaluating the model? Or do you test the entire data set during the test, and then get the indicators in the paper?

**Comment by johanneskiesel** [comment]
Hi,

Sorry for not yet answering you in the other issue. It needs a bit more time from my side, and I could not find that yet.

Regarding your question here:

The only model we trained is that of Meier et al. The other ones we used out-of-the-box. For Meier et al. we use a 10 fold cross validation setup: https://github.com/webis-de/ecir21-an-empirical-comparison-of-web-page-segmentation-algorithms/tree/master/algorithms/meier#train-the-model . If you want to use the same folds, you can find the fold assignment here: https://zenodo.org/record/4337578/files/webis-webseg-20-folds.txt

I hope this answers your question.

Sorry again for being so slow for the other one!

**Comment by Bill-Ren** [comment]
OK, thank you for your patient answer. Because I have been studying this data set recently, I asked a lot of questions. Thank you again for your patient answer.
